lib/ui/notifications.dart

```diff
@@ -17,29 +18,43 @@ GlobalConfiguration cfg = GlobalConfiguration();
 
 class NotificationsWidgetState extends AssistantState<NotificationsWidget> {
   GlobalConfiguration cfg = GlobalConfiguration();
+  var notificationSubList = <NotificationSubscription>[];
+
+  var notificationSub = new NotificationSubscription('Test Keyword');  //TODO: create fetching new notification subscription
```

Comment:
We should use `MailNotifier::getSubscriptions` here, since we have that available now.